sdk/scheduler/src/test/java/com/mesosphere/sdk/http/EndpointUtilsTest.java

```diff
@@ -33,49 +35,73 @@ public void testToEndpoint() {
 
     @Test
     public void testToAutoIpEndpoint() {
-        assertEquals("task.svc.autoip.dcos.thisdcos.directory:5", EndpointUtils.toAutoIpEndpoint("svc", "task", 5, SchedulerConfigTestUtils.getTestSchedulerConfig()));
-        assertEquals("task.pathtosvc.autoip.dcos.thisdcos.directory:5", EndpointUtils.toAutoIpEndpoint("/path/to/svc", "task", 5, SchedulerConfigTestUtils.getTestSchedulerConfig()));
-        assertEquals("task.pathtosvc.autoip.dcos.thisdcos.directory:5", EndpointUtils.toAutoIpEndpoint("path/to/svc", "task", 5, SchedulerConfigTestUtils.getTestSchedulerConfig()));
-        assertEquals("task.pathtosvc-with-dots.autoip.dcos.thisdcos.directory:5", EndpointUtils.toAutoIpEndpoint("path/to/svc.with.dots", "task", 5, SchedulerConfigTestUtils.getTestSchedulerConfig()));
+        assertEquals("task.svc.autoip.tld:5",
+                EndpointUtils.toAutoIpEndpoint("svc", "task", 5, SchedulerConfigTestUtils.getTestSchedulerConfig()));
+        assertEquals("task.pathtosvc.autoip.tld:5",
+                EndpointUtils.toAutoIpEndpoint("/path/to/svc", "task", 5, SchedulerConfigTestUtils.getTestSchedulerConfig()));
+        assertEquals("task.pathtosvc.autoip.tld:5",
+                EndpointUtils.toAutoIpEndpoint("path/to/svc", "task", 5, SchedulerConfigTestUtils.getTestSchedulerConfig()));
+        assertEquals("task.pathtosvc-with-dots.autoip.tld:5",
+                EndpointUtils.toAutoIpEndpoint("path/to/svc.with.dots", "task", 5, SchedulerConfigTestUtils.getTestSchedulerConfig()));
     }
 
     @Test
     public void testToAutoIpEndpointCustomTLD() {
         SchedulerConfig mockSchedulerConfig = SchedulerConfigTestUtils.getTestSchedulerConfig();
-        when(mockSchedulerConfig.getServiceTLD()).thenReturn("what.a.fun.test.tld");
+        when(mockSchedulerConfig.getAutoipTLD()).thenReturn("what.a.fun.test.tld");
```

Comment:
This seems like a duplication of the previous `testToAutoIpEndpoint` test and I don't really see it adding additional functionality considering that we're mocking `getAutoipTLD()` in both cases. We are still not checking the effect of the envvar.